roles/container-engine/docker/templates/docker-options.conf.j2

```diff
@@ -1,12 +1,12 @@
 [Service]
 Environment="DOCKER_OPTS={{ docker_options|default('') }} --iptables={{ docker_iptables_enabled | default('false') }} \
+--exec-opt native.cgroupdriver={{ docker_cgroup_driver | default('systemd') }} \
```

Comment:
I'd prefer to keep default values in the defaults/main.yml and not hide them in various places